pkg/helm/helm.go

```
@@ -248,3 +259,34 @@ func (r *Impl) ChartPull(ref string) error {
 	var buf bytes.Buffer
 	return push.Run(&buf, ref)
 }
+
+// GetResources returns allocated resources for the specified release (their current state)
+func (r *Impl) GetResources(kubeNamespace string, releaseName string) ([]*unstructured.Unstructured, error) {
+	resources := make([]*unstructured.Unstructured, 0)
+	var rel *release.Release
+	var config *action.Configuration
+	var err error
+	var resourceList kube.ResourceList
+	config, err = getConfig(kubeNamespace)
+	if err != nil || config == nil {
+		return resources, err
+	}
+	status := action.NewStatus(config)
+	rel, err = status.Run(releaseName)
+	if err != nil {
+		return resources, err
+	}
+	resourceList, _ = config.KubeClient.Build(bytes.NewBufferString(rel.Manifest), false)
```

Is `_` an ignored error?

---

yes

---

Don't ignore errors :)

---

Committed an error check.